### DISCUSSION TITLE
v0.1.0-beta.9

### DISCUSSION THREAD

**Discussion by philipborg** [philipborg/CubeShuffle#109] [Releases]
## What's Changed
* Build universal macOS tauri build. by @philipborg in https://github.com/philipborg/CubeShuffle/pull/101
* Update dependencies. by @philipborg in https://github.com/philipborg/CubeShuffle/pull/102
* Add dedicated "Done" button to pack picking. by @philipborg in https://github.com/philipborg/CubeShuffle/pull/103
* 74 allow cube sizes which arent divisible by the pack size by @philipborg in https://github.com/philipborg/CubeShuffle/pull/104
* Update CLAP API usage; stop using deprecated API. by @philipborg in https://github.com/philipborg/CubeShuffle/pull/105
* Add normalize.css, making CSS uniform for all web renderers. by @philipborg in https://github.com/philipborg/CubeShuffle/pull/106
* Hide overflow-x on html instead of body. by @philipborg in https://github.com/philipborg/CubeShuffle/pull/107

**Full Changelog**: https://github.com/philipborg/CubeShuffle/compare/v0.1.0-beta.8...v0.1.0-beta.9

<hr /><em>This discussion was created from the release <a href='https://github.com/philipborg/CubeShuffle/releases/tag/v0.1.0-beta.9'>v0.1.0-beta.9</a>.</em>

**Comment by rinaldijuw** [comment]
Tested again on Android this time, and it's perfect! I love the "Done" feature!

**Reply by philipborg** [reply]
Glad to hear and thanks! I will hopefully get some time soon to work a bit more on CubeShuffle, been a bit busy lately. Would love to hit the [0.1.0 milestone](https://github.com/philipborg/CubeShuffle/milestone/1) soon.
